Makefile

```diff
@@ -29,18 +29,19 @@ build: ## Builds the all docker services.
 	docker compose build
 
 run-devserver: # Runs the dev-server
-	docker compose up dev-server -d
+	docker compose up dev-server -d --no-deps
```

Comment:
Why is the `--no-deps` added here?

Reply:
so that no other container get started. since we need to get the backend env to update the backend deps, and dev-server depends on all other containers, so we need to append this --no-deps flag

Reply:
If we're setting up for the first time, then will this throw an error since other containers have not been setup?